Formula/qt-libiodbc.rb

```diff
@@ -0,0 +1,75 @@
+class QtLibiodbc < Formula
+  desc "Qt SQL Database Driver"
+  homepage "https://www.qt.io/"
+  url "https://download.qt.io/official_releases/qt/6.0/6.0.1/submodules/qtbase-everywhere-src-6.0.1.tar.xz"
+  sha256 "8d2bc1829c1479e539f66c2f51a7e11c38a595c9e8b8e45a3b45f3cb41c6d6aa"
+  license all_of: ["GPL-2.0-only", "GPL-3.0-only", "LGPL-2.1-only", "LGPL-3.0-only"]
+
+  depends_on "cmake" => [:build, :test]
+
+  depends_on "libiodbc"
+  depends_on "qt"
+
+  conflicts_with "qt-unixodbc",
+    because: "qt-unixodbc and qt-libiodbc install the same binaries"
+
+  def install
+    cd "src/plugins/sqldrivers" do
+      system "qmake", "--", "ODBC_PREFIX=#{Formula["libiodbc"].opt_prefix}"
+      system "make", "sub-odbc"
+      (share/"qt").install "plugins/"
+    end
+    Pathname.glob(share/"qt/plugins/sqldrivers/*.dylib") do |plugin|
+      system "strip", "-S", "-x", plugin
```

Comment:
Is there no `make install` we can use that already strips the binaries when they're installed?

Reply:
It has, but the prefix is qt’s prefix, and there is no argument to change prefix.